### PR TITLE
fix: fix the preview img size

### PR DESCRIPTION
**Pull request by nova1751**
Fix the preview img size bug.
fix https://github.com/ant-design/ant-design/issues/49235
close #353

### CODE REVIEW

**Comment by nova1751** [comment]
Fix: https://github.com/ant-design/ant-design/discussions/49234
https://github.com/ant-design/ant-design/issues/49235

**Comment by yoyo837** [comment]
这个对antd算不算breaking？

**Comment by nova1751** [comment]
只是bug fix

**Comment by yoyo837** [comment]
测试用例

File: src/Image.tsx [hunk 1]
```diff
@@ -240,8 +240,8 @@ const ImageInternal: CompoundedComponent<ImageProps> = props => {
           mousePosition={mousePosition}
           src={src}
           alt={alt}
-          width={width}
-          height={height}
+          widthInfo={width}
+          heightInfo={height}
```
Comment (crazyair) [inline src/Image.tsx:244]: 加个 imageInfo={{ width, height }}

File: tests/preview.test.tsx [hunk 2]
```diff
@@ -796,13 +796,27 @@ describe('Preview', () => {
...
             return (
               <>
-                <div id="flipY" onClick={() => actions.onFlipY()}>{icons.flipYIcon}</div>
-                <div id="flipX" onClick={() => actions.onFlipX()}>{icons.flipXIcon}</div>
-                <div id="zoomIn" onClick={() => actions.onZoomIn()}>{icons.zoomInIcon}</div>
-                <div id="zoomOut" onClick={() => actions.onZoomOut()}>{icons.zoomOutIcon}</div>
-                <div id="rotateLeft" onClick={() => actions.onRotateLeft()}>{icons.rotateLeftIcon}</div>
-                <div id="rotateRight" onClick={() => actions.onRotateRight()}>{icons.rotateRightIcon}</div>
-                <div id="reset" onClick={() => actions.onReset()}>reset</div>
+                <div id="flipY" onClick={() => actions.onFlipY()}>
```
Comment (crazyair) [inline tests/preview.test.tsx:799]: 撤销？

Comment (nova1751) [reply tests/preview.test.tsx:799]: 这个好像是自动格式化的

**Pull request merged**